notebooks/community/ml_ops/stage7/model_monitoring_custom.ipynb

```diff
@@ -0,0 +1,1267 @@
+{
```

Comment:
<p>Can we make it clear that this is optional and why we do this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>done</p>